types/chargebee__chargebee-js-react-wrapper/index.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for chargebee-js-wrappers 0.2
+// Project: https://www.npmjs.com/package/@chargebee/chargebee-js-react-wrapper
+// Definitions by: Sagar Jain <https://github.com/sagar7993>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Component, CSSProperties } from 'react';
+
+export interface TokenizeAdditionalData {
+    readonly firstName?: string;
+    readonly lastName?: string;
+    readonly billingAddr1?: string;
+    readonly billingAddr2?: string;
+    readonly billingCity?: string;
+    readonly billingState?: string;
+    readonly billingStateCode?: string;
+    readonly billingZip?: string;
+    readonly billingCountry?: string;
+}
+export interface AuthorizeWith3dsAdditionalDataAddress {
+    readonly firstName?: string;
+    readonly lastName?: string;
+    readonly phone?: string;
+    readonly addressLine1?: string;
+    readonly addressLine2?: string;
+    readonly addressLine3?: string;
+    readonly city?: string;
+    readonly state?: string;
+    readonly stateCode?: string;
+    readonly countryCode?: string;
+    readonly zip?: string;
+}
+export interface AuthorizeWith3dsAdditionalData {
+    readonly plan?: string;
+    readonly email?: string;
+    readonly phone?: string;
+    readonly billingAddress?: AuthorizeWith3dsAdditionalDataAddress;
+    readonly customerBillingAddress?: AuthorizeWith3dsAdditionalDataAddress;
+    readonly shippingAddress?: AuthorizeWith3dsAdditionalDataAddress;
+}
+export interface PaymentIntent {
+    readonly id: string;
+    readonly status: 'inited' | 'in_progress' | 'authorized' | 'consumed' | 'expired';
+    /**
+     * The currency code (ISO 4217 format) of the amount used in transaction.
+     */
+    readonly currency_code?: string;
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/chargebee__chargebee-js-react-wrapper/index.d.ts;
  Line: 47;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.